LICENSE

```diff
@@ -1,22 +0,0 @@
-The MIT License (MIT)
```

Comment:
Are you replacing this with anything? I suggest the [Unlicense](http://unlicense.org/).